lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb

```diff
@@ -44,5 +52,49 @@ def self.xcodebuild(options)
         raise "Fail to build targets: #{targets}"
       end
     end
+
+    def self.xcodebuild_archive(options)
+      sdk = options[:sdk] || "iphonesimulator"
+      # targets = options[:targets] || [options[:target]]
+      # platform = PLATFORM_OF_SDK[sdk]
+      target = options[:target]
+
+      cmd = ["xcodebuild"]
+      cmd << "-project" << options[:sandbox].project_path.realdirpath.shellescape
+      # targets.each { |target| cmd << "-target" << target }
+      cmd << "-scheme" << target
+      cmd << "-configuration" << options[:configuration]
+      cmd << "-sdk" << sdk
+      cmd << "-destination" << DESTINATION_OF_SDK[sdk]
+      cmd << "-archivePath" << "#{options[:build_dir]}/#{options[:configuration]}-#{sdk}/#{target}"
+      # unless platform.nil?
+      #   cmd << Fourflusher::SimControl.new.destination(:oldest, platform, options[:deployment_target])
+      # end
+      cmd += options[:args] if options[:args]
+      cmd << "archive"
+      cmd << "2>&1"
+      cmd = cmd.join(" ")
+
+      if !PodPrebuild.config.silent_build?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/grab/cocoapods-binary-cache/blob/b144f30fb41bd34e49fceb072710e5f2c3a3e710/lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb#L78"></span>Style/NegatedIf: Favor <code>unless</code> over <code>if</code> for negative conditions.</td>
    </tr>
  </tbody>
</table>